controllers/k8ssandra/k8ssandracluster_controller_test.go

```diff
@@ -400,13 +395,6 @@ func applyClusterTemplateConfigs(t *testing.T, ctx context.Context, f *framework
 	assert.Equal(dc1Size, dc1.Spec.Size)
 	assert.Equal(dc1.Spec.SuperuserSecretName, superUserSecretName)
 
-	actualConfig, err := gabs.ParseJSON(dc1.Spec.Config)
```

Comment:
I removed this kind of test code. All it does is serialize the config twice, and assert that the first result is equal to the second.